aws-api/src/test/java/com/amplifyframework/api/aws/AWSApiPluginTest.java

```diff
@@ -222,6 +237,63 @@ public void graphQlMutationGetsResponse() throws JSONException, ApiException {
         assertEquals(expectedName, actualResponse.getData().getName());
     }
 
+    /**
+     * Test that owner variable is added to subscription when needed.
+     * @throws JSONException from JSONAssert.assertEquals
+     */
+    @Test
+    public void graphQlSubscriptionAddsOwnerVariable() throws JSONException {
+        GraphQLRequest<OwnerAuth> request = ModelSubscription.onCreate(OwnerAuth.class);
+        plugin.subscribe(request,
+            subscriptionId -> { },
+            response -> { },
+            exception -> { },
+            EmptyAction.instance());
+
+        JSONAssert.assertEquals(Resources.readAsString("request-ownerauth.json"),
+                request.getContent(),
+                true);
+    }
+
+    /**
+     * Verify that the custom owner field variable is added to subscription when needed.
+     * @throws JSONException from JSONAssert.assertEquals
+     */
+    @Test
+    public void graphQLSubscriptionAddsCustomOwnerField() throws JSONException {
+        GraphQLRequest<OwnerAuthCustomField> request = ModelSubscription.onUpdate(OwnerAuthCustomField.class);
+        plugin.subscribe(request,
+            subscriptionId -> { },
+            response -> { },
+            exception -> { },
+            EmptyAction.instance());
```

Comment:
Do we have an `EmptyConsumer` or `NoOpConsumer` somewhere, right now? I think so. If you use the `EmptyAction.instance()`, it would be nice to keep all of these in a similar manner of expression. (Same for above, in this test file, too.)

Reply:
I just found `NoOpConsumer` :). I've updated to use it.